orange/src/main/java/com/hartwig/hmftools/orange/report/tables/ImmuneEscapeTable.java

```diff
@@ -0,0 +1,61 @@
+package com.hartwig.hmftools.orange.report.tables;
+
+import com.hartwig.hmftools.datamodel.immuno.ImmuneEscapeRecord;
+import com.hartwig.hmftools.orange.report.ReportResources;
+import com.hartwig.hmftools.orange.report.util.Cells;
+import com.hartwig.hmftools.orange.report.util.Tables;
+import com.itextpdf.layout.element.Cell;
+import com.itextpdf.layout.element.Table;
+
+import org.apache.logging.log4j.util.Strings;
+import org.jetbrains.annotations.NotNull;
+
+public final class ImmuneEscapeTable
+{
+    @NotNull
+    public static Table build(@NotNull String title, float width, @NotNull ImmuneEscapeRecord immuneEscape,
+            @NotNull ReportResources reportResources, boolean isTumorFail)
+    {
+        Cells cells = new Cells(reportResources);
+        Table table = Tables.createContent(width,
+                new float[] { 2, 1, 3 },
+                new Cell[] { cells.createHeader("Escape Mechanism"), cells.createHeader("Present?"), cells.createHeader(Strings.EMPTY) });
```

Comment:
I see your point, agree "present" is too definitive. I have changed it to "detected?" which I think covers the analysis better. If something is not detected then it could still be present in theory. 

I do think it is nice to display all 6 categories to make it transparant what we actually look for in every report. 

Have also considered updating the datamodel based on your feedback but I think those names are fine, fairly consistent with other characteristics of a tumor we try to capture but with <100% sensitivity.